### PR TITLE
Bug/password input cursor is jumping when typing password fast

### LINKED ISSUE #19004: Password input cursor is jumping when typing password fast
Might be related to https://github.com/status-im/status-mobile/pull/18728 cc @yqrashawn 

**Steps:** 

1. Open login page
2. Try entering password fast
3. See cursor's behaviour

**Actual result:** cursor is jumping back which results in entering wrong symbols

https://github.com/status-im/status-mobile/assets/97245802/2879ed0b-b2ba-4b12-a8da-9565150a04b3

### Additional Information

- Status version: nightly 
- Operating System: Android, iOS

### PR DESCRIPTION
**Pull request by flexsurfer**
fixes  #19004

there is a bug on some Android devices, that when text input is re-rendered text is removed from the native component, so temporary solution is to use default value, i used atom so it won't trigger re-rendering

### CODE REVIEW

File: src/status_im/common/standard_authentication/password_input/view.cljs [hunk 1]
```diff
@@ -21,31 +21,62 @@
...
+                                                                       {:password (security/mask-data
+                                                                                   entered-password)
+                                                                        :error    ""}]
+                                                                      100)))]
     [:<>
      [rn/view {:style {:flex-direction :row}}
       [quo/input
        {:container-style {:flex 1}
         :type            :password
+        :default-value   @default-value
```
Comment (flexsurfer) [inline src/status_im/common/standard_authentication/password_input/view.cljs:72]: this is the only related change, other just slight adjusting according new UI guidelines

**Review by Parveshdhull — APPROVED** [review]

**Review by siddarthkay — APPROVED** [review]

**Review by yqrashawn — APPROVED** [review]

**Comment by pavloburykh** [comment]
@flexsurfer thanx for the PR. Initial issue is fixed and works fine. 
But I noticed some unexpected behaviour of cursor in name input field (IOS only). 

### ISSUE 1 Sometimes cursor jumps back when typing fast in the name input filed (IOS)

reproducing on iPhone X, IOS 16.7.5

**Steps:** 

1. Open onboarding Create profile screen
2. Start typing name fast
3. See cursor's behaviour

**Actual result:** Sometimes cursor jumps back when typing name fast. Also sometimes cursor starts lagging when deleting inout value. 


https://github.com/status-im/status-mobile/assets/97245802/495b97d9-fd5a-426d-ba95-f224d5c89f10

**Comment by flexsurfer** [comment]
most likely it was introduced not in this PR ,could you create a separate issue please?

**Comment by pavloburykh** [comment]
> most likely it was introduced not in this PR ,could you create a separate issue please?

sorry, you are right. I have just checked in another rebased PR and was able to reproduce. This PR is ready for merge.

**Pull request merged**